.gitignore

```diff
@@ -17,3 +17,4 @@ output
 .venv
 .asv
 venv
+.devcontainer
```

Comment:
BTW If you'd like well-setup devcontainer for this project with pyenv python 36 and 310 ready to go, lmk and I can make a separate PR.